android/sdk/src/main/java/org/jitsi/meet/sdk/AudioDeviceHandlerConnectionService.java

```diff
@@ -141,8 +145,9 @@ public void start(AudioModeModule audioModeModule) {
         JitsiMeetLogger.i("Using " + TAG + " as the audio device handler");
 
         module = audioModeModule;
+        context = module.getContext();
```

Comment:
Actually, wouldn't it make more sense to just keep a reference to RNConnectionService? (Again, making sure we set it to null onStop)

Reply:
Changed it